keys.go

```diff
@@ -302,6 +374,47 @@ type ForceOpt struct {
 	Force bool
 }
 
+// ListKeyVersionsOptions struct to add the query parameters for the ListKeyVersions function
+type ListKeyVersionsOptions struct {
+	Limit      *uint32
+	Offset     *uint32
+	TotalCount *bool
+}
+
+// ListKeyVersions gets all the versions of the key resource by specifying ID of the key and/or optional parameters
+// https://cloud.ibm.com/apidocs/key-protect#getkeyversions
+func (c *Client) ListKeyVersions(ctx context.Context, id string, listKeyVersionsOptions *ListKeyVersionsOptions) (*KeyVersions, *uint, error) {
```

Comment:
We can remove the totalCount as separate response because it is already part of Metadata